index.html

```diff
@@ -1,290 +1,428 @@
+    console.log("Hello " + event.payload + "!");
+
+    return "Message received!";
+  });
+
+  events.process();
+                  </pre>
+                    </div>
+                    <div class="carousel-controls">
+                      <label for="img-2" class="prev-slide">
+                        <span>&lsaquo;</span>
+                      </label>
+                      <label for="img-4" class="next-slide">
+                        <span>&rsaquo;</span>
+                      </label>
+                    </div>
+                  </li>
+                  <input type="radio" name="radio-buttons" id="img-4" />
+                  <li class="slide-container">
+                    <div class="slide-image">
+                <pre>
+  <em>// Utilize a shared workspace between jobs in JavaScript</em>
+  const { events, Job } = require("@brigadecore/brigadier");
+
+  events.on("brigade.sh/cli", "exec", async event => {
+    let job1 = new Job("first-job", "debian:latest", event);
+    job1.primaryContainer.workspaceMountPath = "/share";
+    job1.primaryContainer.command = ["bash"];
+    job1.primaryContainer.arguments = ["-c", "echo 'Hello!' > /share/message"];
+    await job1.run();
+
+    let job2 = new Job("second-job", "debian:latest", event);
+    job2.primaryContainer.workspaceMountPath = "/share";
+    job2.primaryContainer.command = ["cat"];
+    job2.primaryContainer.arguments = ["/share/message"];
+    await job2.run();
+  });
+
+  events.process();
+                  </pre>
+                    </div>
+                    <div class="carousel-controls">
+                      <label for="img-3" class="prev-slide">
+                        <span>&lsaquo;</span>
+                      </label>
+                    </div>
+                  </li>
+                  <div class="carousel-dots">
+                    <label for="img-1" class="carousel-dot" id="img-dot-1"></label>
+                    <label for="img-2" class="carousel-dot" id="img-dot-2"></label>
+                    <label for="img-3" class="carousel-dot" id="img-dot-3"></label>
+                    <label for="img-4" class="carousel-dot" id="img-dot-4"></label>
+                  </div>
+                </ul> <!-- /Slides -->
+
+                <ul class="slide-triggers">
+                  <li class="slide-trigger">
+                    <label for="img-1" id="img-dot-1">
+                      <img src="assets/images/ex-icon-01.png" /> Unit tests for a GitHub push
+                    </label>
+                  </li>
+                  <li class="slide-trigger">
+                    <label for="img-2" id="img-dot-2">
+                      <img src="assets/images/ex-icon-07.png" /> Update a CosmosDB database
+                    </label>
+                  </li>
+                  <li class="slide-trigger">
+                    <label for="img-3" id="img-dot-3">
+                      <img src="assets/images/ex-icon-05.png" /> Send a Slack message
```

Comment:
```suggestion
                      <img src="assets/images/ex-icon-05.png" /> Handle a Slack message event
```
(per https://github.com/brigadecore/brigade-www/pull/55#issuecomment-975988830)